### DISCUSSION TITLE
Rollingupdate kubernetes upgrade?

### DISCUSSION THREAD

**Discussion by Trogie** [authelia/authelia#6386] [General]
Hello all,

I have my Traefik/Authelia config up and running on a Google Kubernetes and works very nice.
But when I update my values.yaml and upgrade the helm install then the rollingupdate with default settings does not seem to work: the only authelia pod running gets terminated immediately and then the updated one starts being created. This gives me a Authelia downtime and hence also for all the sites behind the Traefik SSO Authelia middleware...

Anybody who solved this already?

**Comment by james-d-elliott** [comment]
I don't specifically know the answer to this question, I will try to put aside time to look at it but I'm quite busy. I'd suggest checking specifically how the deployment etc is being deployed and ensure the manifest options match the expected behavior as that may shed some light on it.

I suspect it's due to the default values where the rolling update threshold for unavailable container max is set to 1. I am not entirely sure if you can do a proper rolling update with a single pod.
